app/queries/usernameExists.js

```diff
@@ -0,0 +1,6 @@
+const db = require('../db');
+const sql = require('../db/sql.js');
+
+module.exports = async (username) => {
+  return await db.exists(sql`select * from users where username = ${username}`);
```

Comment:
I think it's better that you just return the promise and let the code that uses this decide whether to use `await`. You might want to use it differently